src/source_sender/mod.rs

```diff
@@ -88,18 +90,42 @@ impl SourceSender {
 
     #[cfg(test)]
     pub fn new_test() -> (Self, impl Stream<Item = Event> + Unpin) {
-        let (pipe, recv) = Self::new_with_buffer(4096);
```

Comment:
Interesting. I hope that doesn't mean this will break in the future again. I couldn't see any reason for the two buffers to be differently sized, so I unified them.